lib/grape/endpoint.rb

```diff
@@ -120,6 +120,7 @@ def reset_routes!
     end
 
     def mount_in(router)
+      @router ||= router
```

Comment:
We didn't address mounting an endpoint in multiple places, but eventually we will. So maybe we don't want `@router`?